core/commands/add.go

```diff
@@ -30,9 +30,11 @@ const (
 
 var AddCmd = &cmds.Command{
 	Helptext: cmds.HelpText{
-		Tagline: "Add a file or directory to ipfs.",
+		Tagline: "Add a file to ipfs.",
```

Comment:
We should just make sure that there is a note in the description. I feel that it may have been better before, now that I'm a few months away from where I was when I started this PR.